runtime/battery-station/src/lib.rs

```diff
@@ -53,10 +53,13 @@ use zeitgeist_primitives::{constants::*, types::*};
 use zrml_rikiddo::types::{EmaMarketVolume, FeeSigmoid, RikiddoSigmoidMV};
 #[cfg(feature = "parachain")]
 use {
-    frame_support::traits::{Everything, Nothing},
+    frame_support::traits::{AsEnsureOriginWithArg, Everything, Nothing},
     frame_system::EnsureSigned,
     xcm_builder::{EnsureXcmOrigin, FixedWeightBounds, LocationInverter},
-    xcm_config::XcmConfig,
+    xcm_config::{
+        asset_registry::{CustomAssetProcessor, CustomMetadata},
+        config::{LocalOriginToLocation, XcmConfig, XcmOriginToTransactDispatchOrigin, XcmRouter},
+    },
```

Comment:
It's because of a very bad code style we have to improve. We basically copy the code from `runtime-common` into the actual runtime crates. In `runtime-common` we use variables that are only defined the actual runtime crates. The following code is from `runtime-common`: https://github.com/zeitgeistpm/zeitgeist/blob/354de95323b087cc9d639c447d33dca4d1ca074c/runtime/common/src/lib.rs#L475-L484
It uses `CustomAssetProcessor`, so we have to import it.